CMSgov/eAPD
# Tbolt/4164 Fixes issue where unable to add duplicate state admin certifications

Resolves #4164 

### Description
When adding a state admin letter, then deleting it, then re-adding it again, the system would not allow it to be re-added. A 400 error was returned in the console.

### Significant changes or possible side effects
No other significant changes or side effects are introduced

### Automated test cases written
Cypress test added to verify a document can be re-added after deletion

### Steps to manually verify this change

1. Login as fedadmin, add a state admin certification
2. Delete that certification
3. Re-add it with the same Name / Email / FFY
4. Verify the system allows it

### This pull request is ready to review when


### This pull request can be merged when
